pkgs/applications/editors/emacs/build-support/wrapper.nix

```
@@ -164,6 +164,7 @@ runCommand (lib.appendToName "with-packages" emacs).name
           # buffer.
           rm -f $siteStart $siteStartByteCompiled $subdirs $subdirsByteCompiled
           cat >"$siteStart" <<EOF
+          ;;; -*- lexical-binding: t -*-
```

```suggestion
          ;; -*- lexical-binding: t; -*-
```
nit: `add-file-local-variable-prop-line` generates this.  Feel free to ignore.